java/hiro-client/src/main/java/co/arago/hiro/client/rest/DefaultHiroClient.java

```diff
@@ -328,7 +329,6 @@ public void getEventStream(Map<String, String> requestParameters, Listener<Strin
         + StringUtils.join(HiroCollections.newList(API_PREFIX, EVENT_STREAM_VERSION, EVENT_STREAM_SUFFIX), "/")
         + prepareEventStreamParams(requestParameters));
       final ClientUpgradeRequest clientUpgradeRequest = new ClientUpgradeRequest();
-      if (LOG.isLoggable(debugLevel)) {
         StringBuilder sb = new StringBuilder();
```

Comment:
here the old logic is better: it's in-efficient to compose complex log message if logging level
will discard it anyway